source/localizable/object-model/computed-properties-and-aggregate-data.md

```diff
@@ -27,7 +27,22 @@ and fire observers when any of the following events occurs:
 3. An item is removed from the `todos` array.
 4. The `todos` property of the component is changed to a different array.
 
-In the example above, the `remaining` count is `1`:
+Sidenote: ember provides the built-in macro `computed.filterBy`, which is syntax
```

Comment:
can you rephrase "built-in macro" and "syntax sugar"? at least while we don't have a good glossary with linkage. "provides a built-in computed property short hand" or something along those lines.